Automattic/wp-calypso
# SMP: center text using flexbox instead of lineheight

#### Proposed Changes

* Fix the site URL CSS to avoid clipping the text

**List**
| Before | After |
|---|---|
| ![before](https://user-images.githubusercontent.com/779993/185989777-269733ef-4939-43cc-9dd0-757a54573670.png)  |  ![after](https://user-images.githubusercontent.com/779993/185990029-8dcd57da-904b-4450-b628-2e2de2d7e0fe.png) |



**Tiles**
| Before | After |
|---|---|
| ![before](https://user-images.githubusercontent.com/779993/185989562-47ec20bc-0dad-4a9f-bc47-f3ac41231fd0.png)  | ![after](https://user-images.githubusercontent.com/779993/185989544-410df26a-df73-4178-98d7-2cf9e0a1e8ed.png)  |




#### Testing Instructions



* Go to `/sites-dashboard`.
* Find a sites with `g`s  in their URL.
* Observe every letter is visible.

#### Pre-merge Checklist






Closes #66758
